assets/js/event-providers/popup-maker.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Site Kit by Google, Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+document.addEventListener( 'DOMContentLoaded', () => {
+	const { jQuery } = global;
+	// eslint-disable-next-line no-undef
+	if ( ! jQuery || ! PUM ) {
+		return;
+	}
+
+	// eslint-disable-next-line no-undef
+	PUM.hooks.addAction( 'pum.integration.form.success', function () {
+		global.gtag( 'event', 'submit_lead_form' );
```

Comment:
Also, the IB doesn't specifically mention the `event_category` or `event_label` fields, but for parity with other plugins we could pass:

`event_category`: `Popup Makers`
`event_label`: ${popup_name}

What do you think?

Reply:
That is not necessary, these two parameters are only sent in CF7, since they are passed and included in their doc, so it is an extra, a bonus. By default we are only sending event name as planned integration.